src/components/ebay-listbox/test/mock/index.js

```diff
@@ -12,6 +12,15 @@ exports.Basic_3Options = {
     })),
 };
 
+exports.Basic_3Options_FirstSelected = {
```

Comment:
would it be helpful to add a test case where none of the options are selected?

Reply:
I have one on server. It checks if no items are selected.